tests/test_add_area.py

```diff
@@ -49,14 +54,67 @@ def _add_area(area, mb, browser):
 
 
 def test_add_area(mb_client, browser, reset_db, area_og):
+    area_types = {
+        "1": "Country",
+        "2": "Subdivision",
+        "3": "City",
+        "4": "Municipality",
+        "5": "District",
+        "6": "Island",
+        "7": "County",
+    }
+
+    link_type_ids = {"wikidata": "358", "geonames": "713"}
+
     try:
         area_mbid = _add_area(area_og, mb_client, browser)
-        assert area_mbid is not None, "Area MBID is None"
+        posted_data = utils.get_entity_json(area_mbid, "area")
+
+        assert area_mbid is not None, "Area MBID is None"  # area added?
+        assert (
+            posted_data["id"] == area_mbid
+        ), "Area MBID is incorrect"  # area MBID correct?
+
+        assert (
+            posted_data["name"] == area_og["name"]
+        ), "Area name is incorrect"  # area name correct?
+
+        assert (
+            posted_data["disambiguation"] == area_og["comment"]
+        ), "Area disambiguation is incorrect"  # area comment correct?
+
+        assert (
+            posted_data["type"] == area_types[area_og["type_id"]]
+        ), "Area type is incorrect"  # area type correct?
```

Comment:
FWIW I don't think we need the comments, they mostly reword the test failure label.